src/verifiable/signatures.ts

```diff
@@ -118,6 +118,32 @@ export function signWithKeys(
     })
 }
 
+/**
+ * Verifies a PayID using the verified addresses within the PayID.
+ *
+ * @param toVerify - The PayID (as a json or as a parsed PaymentInformation).
+ *
+ * @returns True if verified.
+ */
+export function verifyPayId(toVerify: string | PaymentInformation): boolean {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- because JSON
```

Comment:
What exactly is causing this? Is there any way we can "solve" it so that we don't have to disable this rule in various places?

Reply:
I couldn't find a way to do this without upsetting the linter. As I understand it, it is essentially an unsafe operation because I am going from an untyped JSON object to a typed interface. B